prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -1253,6 +1253,25 @@ periodics:
       - 'GCP_CLUSTER=multi-repo-8-standard-regular-gitlab'
 
 ### miscellaneous jobs ###
+- <<: *config-sync-ci-job
+  name: multi-repo-kind
```

Comment:
I think these will also need the `preset-kind-volume-mounts` label

Reply:
DONe